branchout

```diff
@@ -16,15 +17,23 @@ function branchout_set-config() {
   rm "${BRANCHOUT_HOME}/branchoutrc.tmp"
 }
 
+function branchout_ensure_config() {
+  ensureConfigValue "${1}" "${1}"
+}
+
+function branchout_ensure() {
+  ensureValue "${1}" "${1}"
+}
+
 function branchout_get() {
-  grep "^${1}=" "${BRANCHOUT_FILE}" | sed -e 's,^.*="\(.*\)"$,\1,'
+  grep "^${1}=" "${BRANCHOUT_DIRECTORY}/${BRANCHOUT_FILE}" | sed -e 's,^.*="\(.*\)"$,\1,'
 }
 
 function branchout_set() {
-  grep -v "^${1}=" "${BRANCHOUT_FILE}" > "${BRANCHOUT_FILE}".tmp && true
-  echo "${1}=\"${2}\"" >> "${BRANCHOUT_FILE}.tmp"
-  sort "${BRANCHOUT_FILE}.tmp" > "${BRANCHOUT_FILE}"
-  rm "${BRANCHOUT_FILE}.tmp"
+  grep -v "^${1}=" "${BRANCHOUT_DIRECTORY}/${BRANCHOUT_FILE}" > "${BRANCHOUT_DIRECTORY}/${BRANCHOUT_FILE}".tmp && true
+  echo "${1}=\"${2}\"" >> "${BRANCHOUT_DIRECTORY}/${BRANCHOUT_FILE}.tmp"
+  sort "${BRANCHOUT_DIRECTORY}/${BRANCHOUT_FILE}.tmp" > "${BRANCHOUT_DIRECTORY}/${BRANCHOUT_FILE}"
+  rm "${BRANCHOUT_DIRECTORY}/${BRANCHOUT_FILE}.tmp"
```

Comment:
I don't see this in the commit comment, at least, not explicitly, why is relative not good enough now?

Reply:
this fixes issue #29 and is part of this commit because the tests prove it is working correctly